core_backend/app/routers/manage_content.py

```diff
@@ -27,20 +21,6 @@
 logger = setup_logger()
 
 
-class QdrantPayload(BaseModel):
-    """Content payload for qdrant"""
-
-    # Ensure len("*{title}*\n\n{text}") <= 1600
-    content_title: Annotated[str, StringConstraints(max_length=150)]
-    content_text: Annotated[str, StringConstraints(max_length=1446)]
```

Comment:
We need these length constraints. Can you add them to the ContentDB orm along with the comment?